browser/ephemeral_storage/blob_url_browsertest.cc

```diff
@@ -301,6 +302,22 @@ IN_PROC_BROWSER_TEST_F(BlobUrlPartitionEnabledBrowserTest,
   TestBlobsArePartitioned();
 }
 
+class BlobUrlPartitionEnabledWithoutSiteIsolationBrowserTest
+    : public BlobUrlPartitionEnabledBrowserTest {
+ public:
+  BlobUrlPartitionEnabledWithoutSiteIsolationBrowserTest() = default;
+
+  void SetUpCommandLine(base::CommandLine* command_line) override {
+    BlobUrlPartitionEnabledBrowserTest::SetUpCommandLine(command_line);
+    command_line->AppendSwitch(switches::kDisableSiteIsolation);
```

Comment:
this only simulates Android behavior. We need to actually enable all ES-related tests on Android: https://github.com/brave/brave-browser/issues/29019